.eslintrc.yml

```diff
@@ -10,25 +10,25 @@ parserOptions:
     experimentalObjectRestSpread: true
     jsx: true
   sourceType: module
-plugins:
-  - mocha
-  - react
 rules:
-  no-console:
+  eqeqeq:
     - error
-    - allow:
-      - warn
-      - error
   indent:
     - error
-    - 4
+    - 2
   linebreak-style:
     - error
     - unix
+  no-console:
+    - error
+    - allow:
+      - warn
+      - error
+  no-var:
+    - error
   quotes:
     - error
-    - double
-  react/jsx-uses-vars: 1
```

Comment:
we don't have `react` in this library, do we need this configuration?

Reply:
sure we don't need it and I have removed it from eslint file